datafusion/core/src/physical_plan/file_format/file_stream.rs

```diff
@@ -179,6 +227,14 @@ impl<F: FileOpener> FileStream<F> {
                     partition_values,
                 } => match ready!(reader.poll_next_unpin(cx)) {
                     Some(result) => {
+                        if let Some(instant) =
```

Comment:
Here will add first one batch processing time in total scanning time, I think it's acceptable🤔